### PR TITLE
chore: pkg imported more than once

### PR DESCRIPTION
**Pull request by testwill**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
package  is being imported more than once
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: simulator/scheduler/storereflector/storereflector.go
Comment (196Ikuchil) [inline simulator/scheduler/storereflector/storereflector.go:1]: L83
```
klog.ErrorS(nil, "Cannot convert to *corev1.Pod", "obj", newObj)
```

**Review by 196Ikuchil — COMMENTED** [review]
Hi, @testwill !
Thank you for your contribution!
And thanks to your fixing, I noticed a small mistake of me.
So, could you please fix it while you’re at it?

**Review by sanposhiho — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**